acceptance/suites/pre_suite/puppet3_compat/70_install_puppet.rb

```diff
@@ -0,0 +1,22 @@
+# Agent running on the master is current, not legacy.
+legacy_agents = agents.reject { |agent| agent == master }
+
+step "Install Legacy Puppet Agents."
+legacy_agents.each do |host|
+  platform = host.platform
+
+  puppet_version = ENV['PUPPET_LEGACY_VERSION']
+  if not puppet_version
+    fail "PUPPET_LEGACY_VERSION is not set, e.g. '3.7.5'"
+  end
+  install_package host, 'puppet', puppet_version
+end
+
+step "Install Puppet Server."
+make_env = {
+  "prefix"  => "/usr",
+  "confdir" => "/etc/",
+  "rundir"  => "/var/run/puppetserver",
```

Comment:
I'm not entirely sure what this is for actually, it was cargo-culted from the FOSS pre-suite.

I think we can eliminate the make_env actually, so I'll try that now.